Example/Custom Integration/SEPADebitExampleViewController.m

```diff
@@ -0,0 +1,85 @@
+//
+//  SEPADebitExampleViewController.m
+//  Custom Integration
+//
+//  Created by Cameron Sabol on 10/7/19.
+//  Copyright © 2019 Stripe. All rights reserved.
+//
+
+#import "SEPADebitExampleViewController.h"
+
+#import "MyAPIClient.h"
+
+@interface SEPADebitExampleViewController ()
+
+@end
+
+@implementation SEPADebitExampleViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.title = @"SEPA Debit";
+
+    [self.payButton setTitle:@"Pay with SEPA Debit" forState:UIControlStateNormal];
+    [self.payButton sizeToFit];
+
+    UILabel *mandateAuthLabel = [[UILabel alloc] init];
+    mandateAuthLabel.numberOfLines = 0;
+    mandateAuthLabel.font = [UIFont preferredFontForTextStyle:UIFontTextStyleCallout];
+    mandateAuthLabel.textAlignment = NSTextAlignmentCenter;
+    mandateAuthLabel.text = @"By providing your IBAN and confirming this payment, you are authorizing EXAMPLE COMPANY NAME and Stripe, our payment service provider, to send instructions to your bank to debit your account and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.";
```

Comment:
as a user reading this, I'd be wondering what the requirements for this text are.  Not sure if the docs will cover that or if we should add a comment here